src/transport-manager.ts

```diff
@@ -216,6 +223,11 @@ export class DefaultTransportManager implements TransportManager, Startable {
             })
         })
         listener.addEventListener('close', () => {
```

Comment:
should we be removing these eventListeners once they are closed as well?

Reply:
I don't think so because the listener is created and returned by the transport but not retained.